forms-flow-web/src/components/Form/constants/ClientTable.js

```diff
@@ -289,54 +286,20 @@ function ClientTable() {
       </LoadingOverlay>
 
       {formData.length ? (
-        <div className="d-flex justify-content-between align-items-center flex-column flex-md-row">
-          <div className="d-flex align-items-center">
-            <span className="me-2"> {t("Rows per page")}</span>
-            <Dropdown data-testid="page-limit-dropdown">
-              <Dropdown.Toggle
-                variant="light"
-                id="dropdown-basic"
-                data-testid="page-limit-dropdown-toggle"
-              >
-                {pageLimit}
-              </Dropdown.Toggle>
-
-              <Dropdown.Menu>
-                {pageOptions.map((option, index) => (
-                  <Dropdown.Item
-                    key={index}
-                    type="button"
-                    onClick={() => {
-                      onSizePerPageChange(option.value);
-                    }}
-                    data-testid={`page-limit-dropdown-item-${option.value}`} 
-                  >
-                    {option.text}
-                  </Dropdown.Item>
-                ))}
-              </Dropdown.Menu>
-            </Dropdown>
-            <span className="ms-2">
-              {t("Showing")} {(limit * pageNo) - (limit - 1)} {t("to")}{" "}
-              {limit * pageNo > totalForms ? totalForms : limit * pageNo}{" "}
-              {t("of")} {totalForms} {t("Results")}
-            </span>
-          </div>
-          <div className="d-flex align-items-center">
-            <Pagination
-              activePage={pageNo}
-              itemsCountPerPage={limit}
-              totalItemsCount={totalForms}
-              pageRangeDisplayed={5}
-              itemClass="page-item"
-              linkClass="page-link"
-              onChange={handlePageChange}
-            />
-          </div>
-        </div>
-      ) : (
-        ""
-      )}
+        <table className="table">
+          <tfoot>
+          <TableFooter
+            limit={limit}
+            activePage={pageNo}
```

Comment:
in every case @arun-s-aot  need to show some message if there is no data in that table ?

Reply:
Yes. Fahad's PR will solve that. @shuhaib-aot .